apps/chat/channels.py

```diff
@@ -331,8 +331,12 @@ def start_conversation(self):
         self.experiment_session.update_status(SessionStatus.ACTIVE)
         # This is technically the start of the conversation
         if self.experiment.seed_message:
-            bot_response = self._generate_response_for_user(self.experiment.seed_message)
-            self.send_message_to_user(bot_response)
+            self._send_seed_message()
+
+    def _send_seed_message(self):
+        topic_bot = self.bot or get_bot(self.experiment_session, experiment=self.experiment)
+        bot_response = topic_bot.process_input(user_input=self.experiment.seed_message, save_input_to_history=False)
```

Comment:
Is the seed message a message that 1. OCS sends to the bot on the user's behalf or 2. OCS sends to the user on the bot's behalf? From here, it looks like 1. We should probably add some help text to make that clearer

Reply:
yes, it is 1 - a message we send to the bot on the user's behalf to generate an initial response.